scipio/src/dma_file.rs

```diff
@@ -466,3 +491,182 @@ fn fallback_drop_closes_the_file() {
         });
     }
 }
+
+#[test]
+fn file_create_close() {
+    let ex = LocalExecutor::new(None).unwrap();
```

Comment:
take a look at the `test_executor` macro in lib.rs. It is supposed to make this a little bit more ergonomic.
While that's not a big deal, in this case you have your own helper function to create the directory that youy always call. So you can wrap that into a macro that lives in this file and don't have to repeat yourself.

Not a blocker

Reply:
Fixed in 8981a5a